<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="120" started-at="2018-06-29T07:00:12Z" finished-at="2018-06-29T07:00:12Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="120" started-at="2018-06-29T07:00:12Z" finished-at="2018-06-29T07:00:12Z">
      <class name="readFile.TestCases">
        <test-method status="PASS" signature="initialize()[pri:0, instance:readFile.TestCases@12cc05a]" name="initialize" is-config="true" duration-ms="41" started-at="2018-06-29T07:00:12Z" finished-at="2018-06-29T07:00:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="testCaseToValidateNodeName()[pri:0, instance:readFile.TestCases@12cc05a]" name="testCaseToValidateNodeName" duration-ms="16" started-at="2018-06-29T07:00:12Z" finished-at="2018-06-29T07:00:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCaseToValidateNodeName -->
        <test-method status="FAIL" signature="testCaseToValidatePrice()[pri:0, instance:readFile.TestCases@12cc05a]" name="testCaseToValidatePrice" duration-ms="4" started-at="2018-06-29T07:00:12Z" finished-at="2018-06-29T07:00:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<44> but was:<44.95>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<44> but was:<44.95>
at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.failNotEquals(Assert.java:834)
at org.junit.Assert.assertEquals(Assert.java:118)
at org.junit.Assert.assertEquals(Assert.java:144)
at readFile.TestCases.testCaseToValidatePrice(TestCases.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCaseToValidatePrice -->
        <test-method status="PASS" signature="testCaseToValidationAuthorName()[pri:0, instance:readFile.TestCases@12cc05a]" name="testCaseToValidationAuthorName" duration-ms="2" started-at="2018-06-29T07:00:12Z" finished-at="2018-06-29T07:00:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCaseToValidationAuthorName -->
      </class> <!-- readFile.TestCases -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
